event/event.go

```diff
@@ -8,26 +8,35 @@ import (
 	"go.uber.org/zap/zapcore"
 
 	uuid "github.com/satori/go.uuid"
+	"github.com/serverless/event-gateway/internal/zap"
 )
 
 // Event is a default event structure. All data that passes through the Event Gateway is formatted as an Event, based on
 // this schema.
 type Event struct {
-	Type       Type        `json:"event"`
-	ID         string      `json:"id"`
-	ReceivedAt uint64      `json:"receivedAt"`
-	Data       interface{} `json:"data"`
-	DataType   string      `json:"dataType"`
+	Namespace          string `json:"namespace"`
+	EventType          Type   `json:"event-type"`
+	EventTypeVersion   string `json:"event-type-version"`
+	CloudEventsVersion string `json:"cloud-events-version"`
+	Source             struct {
+		SourceType string `json:"source-type"`
+		SourceID   string `json:"source-id"`
+	} `json:"source"`
+	EventID     string                 `json:"event-id"`
+	EventTime   uint64                 `json:"event-time"`
+	ContentType string                 `json:"content-type"`
```

Comment:
There's a `schema-url` property that's been added to the spec. It's optional and must be a URI.